### DISCUSSION TITLE
Be able to delete old versions of its packages

### DISCUSSION THREAD

**Discussion by pioupia** [npm/feedback#286] [General]
Hello,
I would like to know if it is possible to add the possibility to remove its old versions of npm package. This will take you less memory space, and allow us to eliminate versions that had flaws/ consumed too many resources...

**Comment by MylesBorins** [comment]
@pioupia please refer to our [unpublish policy](https://www.npmjs.com/policies/unpublish). We have to be very careful about allowing unpublishing as it can cause ecosystem breakages.  Below outlines the conditions necessary to allow you to unpublish an old version of a package. If your package verion does not meet those conditions you cannot unpublish it.

> Regardless of how long ago a package was published, you can unpublish a package that:
> 
> * no other packages in the npm Public Registry depend on
> * had less than 300 downloads over the last week
> * has a single owner/maintainer
